### DISCUSSION TITLE
RAKwireless WisBlock RAK4631 as a RNode?

### DISCUSSION THREAD

**Discussion by jacqueszsx** [markqvist/Reticulum#513] [General]
Hi is there anyone looking at RAK www.rakwireless.com "WisBlock" modularity and the RAK4631 (Nordic nRF52840) as a RNode?. An example the WisMeshPocket https://www.youtube.com/watch?v=PyWYncggmcg

**Comment by cobraPA** [comment]
I don't personally know the full status, but support was recently added: https://github.com/markqvist/Reticulum/pull/502

**Reply by jacqueszsx** [reply]
Thank you for link to #502

**Reply by markqvist** [reply]
Check also the Community Edition firmware branch: https://github.com/liberatedsystems/RNode_Firmware_CE

That's where all the RNode firmware development is currently happening!

**Reply by jacqueszsx** [reply]
Thank you Mark
